### PR TITLE
modtool: Sequence Completer for python3 branch

### PR DESCRIPTION
**Pull request by swap-nil7**
I have updated the Sequence Completer added by @marcusmueller for python3 compatibility.
Key update: `filter` returns an iterator rather than a list if a list is passed in py3k.

### CODE REVIEW

File: gr-utils/python/modtool/modtool_add.py [hunk 1]
```diff
@@ -73,17 +73,21 @@ def setup(self, options):
 
         self._info['blocktype'] = options.block_type
         if self._info['blocktype'] is None:
-            # Print(list out of blocktypes to user for reference)
+            # Print list out of blocktypes to user for reference
```
Comment (marcusmueller) [inline gr-utils/python/modtool/modtool_add.py]: Delete this line

File: gr-utils/python/modtool/modtool_add.py [hunk 2]
```diff
@@ -73,17 +73,21 @@ def setup(self, options):
...
+                    self._info['blocktype'] = input("Enter block type: ")
+                    if self._info['blocktype'] not in self._block_types:
+                        print('Must be one of ' + str(self._block_types))
+
         # Allow user to specify language interactively if not set
         self._info['lang'] = options.lang
         if self._info['lang'] is None:
-            while self._info['lang'] not in ['c++', 'cpp', 'python']:
-                self._info['lang'] = input("Language (python/cpp): ")
+            language_candidates = ('cpp', 'python')
```
Comment (marcusmueller) [inline gr-utils/python/modtool/modtool_add.py]: did I or did you omit `'c++'`? If I did it, I can't remember the rationale. If you did it: why is there no `c++`?

**Comment by swap-nil7** [comment]
Currently removed SequenceCompleter from `modtool_rm.py` because of no candidates, but they're there in the `master` branch. 
Done with the mentioned changes!

**Comment by swap-nil7** [comment]
Closing the PR since the work is already there in #1873.

**Pull request closed without merging**